pyvista/plotting/camera.py

```diff
@@ -618,3 +644,70 @@ def copy(self):
             setattr(new_camera, attr, value)
 
         return new_camera
+
+    def tight(self, padding=0, adjust_render_window=True):
+        """Adjust the camera position so that the actors fill the entire renderer.
+
+        Parameters
+        ----------
+        padding : int, optional
```

Comment:
Can this be float?

Reply:
It's absolutely a float and I found the implementation was really a relative percent pad, not a pixel pad. This is what I get for not testing it. Fixed in 91464f77d4791d8395b51c1de8299f05a4694fa8.